icu4c/source/i18n/currpinf.cpp

```diff
@@ -298,12 +298,12 @@ CurrencyPluralInfo::setupCurrencyPluralPattern(const Locale& loc, UErrorCode& st
         const char* pluralCount;
         while ((pluralCount = keywords->next(NULL, ec)) != NULL) {
             if ( U_SUCCESS(ec) ) {
-                int32_t ptnLen;
+                int32_t ptnLength;
```

Comment:
I think some of these changes might conflict with some white-space changes that I have in another PR, but that should be fine. It will be a good test for handling merge conflicts. :)